blockchains/erc20/erc20_worker.js

```diff
@@ -24,17 +25,45 @@ class ERC20Worker extends BaseWorker {
   async init(exporter) {
     this.lastConfirmedBlock = await this.web3.eth.getBlockNumber() - constants.CONFIRMATIONS;
     setGlobalTimestampManager(exporter);
+
+    if (constants.EXPORT_BLOCKS_LIST) {
+      this.blocksList = await initBlocksList();
+    }
+  }
+
+  getBlocksListInterval() {
```

Comment:
Minor. I would have made this a stand alone function operating on two arguments. This makes reuse in other classes easier.